anki-editor.el

```diff
@@ -616,6 +632,18 @@ see `anki-editor-insert-note' which wraps this function."
    (t
     (anki-editor--update-note note))))
 
+(defun anki-editor--calc-note-hash (note)
+  "Calculate an md5 hash of the contents of NOTE."
+  (secure-hash
+   'md5
+   (mapconcat #'prin1-to-string
+              (mapcar (lambda (f) (funcall f note))
+                      (list #'anki-editor-note-id
+                            #'anki-editor-note-model
+                            #'anki-editor-note-deck
+                            #'anki-editor-note-fields
+                            #'anki-editor-note-tags)))))
+
```

Comment:
This is a bit of an edge-case, but we should perhaps introduce at least a variable to control this behaviour. Sometimes, the export result would change even when the content hasn't (e.g., exporting the same code but with a different colour scheme).